app/views/invitation/show.html.haml

```diff
@@ -1,12 +1,12 @@
 .stripe.reverse
   .row
-    .large-12.columns
-      %h2
+    .columns
+      %h1
```

Comment:
in the world of foundation this `large-12` class should still be there right?

Reply:
yep you're right, will add that in, thanks!